arhex:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Position Controllers --------------------------------------
  torso_position_controller:
    type: effort_controllers/JointPositionController
    joint: torso
    pid: {p: 50, i: 0.01, d: 0.1}
  shoulder_1_position_controller:
    type: effort_controllers/JointPositionController
    joint: shoulder_1
    pid: {p: 50, i: 0.01, d: 0.1}
  shoulder_2_position_controller:
    type: effort_controllers/JointPositionController
    joint: shoulder_2
    pid: {p: 50, i: 0.01, d: 0.1}
  elbow_position_controller:
    type: effort_controllers/JointPositionController
    joint: elbow
    pid: {p: 50, i: 0.01, d: 0.1}
  wrist_1_position_controller:
    type: effort_controllers/JointPositionController
    joint: wrist_1
    pid: {p: 50, i: 0.01, d: 0.1}
  wrist_2_position_controller:
    type: effort_controllers/JointPositionController
    joint: wrist_2
    pid: {p: 50, i: 0.01, d: 0.1}
  wrist_3_position_controller:
    type: effort_controllers/JointPositionController
    joint: wrist_3
    pid: {p: 50, i: 0.01, d: 0.1}
  ef_1_position_controller:
    type: effort_controllers/JointPositionController
    joint: ef_1
    pid: {p: 50, i: 0.01, d: 0.1}
  ef_2_position_controller:
    type: effort_controllers/JointPositionController
    joint: ef_2
    pid: {p: 50, i: 0.01, d: 0.1}
  L_FR_position_controller:
    type: effort_controllers/JointPositionController
    joint: L_FR
    pid: {p: 50, i: 0.01, d: 0.1}
  L_MR_position_controller:
    type: effort_controllers/JointPositionController
    joint: L_MR
    pid: {p: 50, i: 0.01, d: 0.1}
  L_RR_position_controller:
    type: effort_controllers/JointPositionController
    joint: L_RR
    pid: {p: 50, i: 0.01, d: 0.1}
  L_RL_position_controller:
    type: effort_controllers/JointPositionController
    joint: L_RL
    pid: {p: 50, i: 0.01, d: 0.1}
  L_ML_position_controller:
    type: effort_controllers/JointPositionController
    joint: L_ML
    pid: {p: 50, i: 0.01, d: 0.1}
  L_FL_position_controller:
    type: effort_controllers/JointPositionController
    joint: L_FL
    pid: {p: 50, i: 0.01, d: 0.1}
